doc/source/data/dataset.rst

```diff
@@ -36,8 +36,22 @@ Data Loading and Preprocessing for ML Training
 Ray Datasets are designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
 Compared to other loading solutions, Datasets are more flexible (e.g., can express higher-quality `per-epoch global shuffles <examples/big_data_ingestion.html>`__) and provides `higher overall performance <https://www.anyscale.com/blog/why-third-generation-ml-platforms-are-more-performant>`__.
 
-Ray Datasets are not intended as a replacement for more general data processing systems.
-:ref:`Learn more about how Ray Datasets work with other ETL systems <datasets-ml-preprocessing>`.
+Datasets is not intended as a replacement for more general data processing systems.
+Its utility is as the last-mile bridge from ETL pipeline outputs to distributed applications and libraries in Ray:
```

Comment:
```suggestion

```

Reply:
If you're comfortable being more opinionated about this guidance. Think of how you would tell a friend who is using Datasets.

Also, do we need to explain what we mean by `last-mile`? I thought that might be a term that we coined internally. If not, no need to explain.

Reply:
Hmm...really not sure how I'd describe Datasets' utility in a simple but precise way.

Maybe something like "Datasets moves data from storage to distributed Ray applications. It also lets you transform data into representations that are more suitable for models."?

Reply:
Can we remove "Its utility is as the last-mile bridge from ETL pipeline outputs to distributed applications and libraries in Ray:".

This is pretty much repeated from above sentence: "Use Datasets for the last-mile bridge from ETL pipeline outputs to distributed applications and libraries in Ray."

We can add a link to "User Guides" here, e.g. "Check out Datasets User Duides for the features it supports".